openvdb/doc/changes.txt

```diff
@@ -6,6 +6,15 @@
 @par
 <B>Version 6.0.1</B> - <I>In development</I>
 
+@par
+New features:
+- Added new @vdblink::QuatTraits QuatTraits@endlink,
+  @vdblink::MatTraits MatTraits@endlink and
+  @vdblink::ValueTraits ValueTraits@endlink type traits to complement
+  @vdblink::VecTraits VecTraits@endlink.
+- Added an @vdblink::IsSpecializationOf @endlink helper metafunction for
```

Comment:
The link should read "`@vdblink::IsSpecializationOf IsSpecializationOf@endlink`".

Reply:
@pcucka - Done, thanks!

I also added a CI job to automate checking the Doxygen - https://github.com/AcademySoftwareFoundation/openvdb/pull/335